srcd/superset/assets/src/uast/stylesheets/UASTModal.less

```diff
@@ -1,9 +1,22 @@
 // make modal window "fullscreen"
 .modal-lg .modal-body {
-    height: ~'calc(100vh - 150px)';
+  height: ~"calc(100vh - 150px)";
 
-    .uast-viewer {
-        height: 100%;
-        overflow-y: auto;
-    }
-}
\ No newline at end of file
+  .uast-viewer {
+    height: 100%;
+    overflow-y: auto;
+  }
+}
+
+.modal-body {
+  font-family: Source Code Pro, Monaco, Consolas, Courier New, monospace;
```

Comment:
This PR introduces a new `Source Code Pro` declaration:
- [src/uast/stylesheets/UASTModal.less#L12](https://github.com/src-d/sourced-ui/blob/code-mirror-modal/srcd/superset/assets/src/uast/stylesheets/UASTModal.less#L12)
  ```less
  .modal-body {font-family: "Source Code Pro", Monaco, Consolas, "Courier New", monospace;}
  ```

We already have:
- [src/uast/codemirror_override.less#L43](https://github.com/src-d/sourced-ui/blob/code-mirror-modal/srcd/superset/assets/src/uast/codemirror_override.less#L43)
  ```less
  .CodeMirror * {font-family: "Source Code Pro", Monaco, Consolas, "Courier New", monospace;}
  ```
- [stylesheets/less/custom-brand.less#L90](https://github.com/src-d/sourced-ui/blob/code-mirror-modal/srcd/superset/assets/stylesheets/less/custom-brand.less#L90)
  ```less
  .ace_editor {font-family: "Source Code Pro", Monaco, Consolas, "Courier New", monospace;}
  ```

I wonder if it would be a good idea to use a new shared variable in `variables-override.less`, as we do at:
- [stylesheets/less/variables-override.less#L48](https://github.com/src-d/sourced-ui/blob/code-mirror-modal/srcd/superset/assets/stylesheets/less/variables-override.less#L48)
  ```less
  @font-family-sans-serif: "Source Sans Pro", "Helvetica Neue", Arial, sans-serif;
  ```

Reply:
_(aside)_

The variant `Source Sans Pro` is currently being used in a couple of places, using its hardcoded name in different ways instead of using `@font-family-sans-serif` shared variable:
- [src/uast/override.less](https://github.com/src-d/sourced-ui/blob/code-mirror-modal/srcd/superset/assets/src/uast/override.less) 7 times
- [stylesheets/less/custom-brand.less](https://github.com/src-d/sourced-ui/blob/code-mirror-modal/srcd/superset/assets/stylesheets/less/custom-brand.less) 2 times

This other case could be considered in a separated PR, so I can open a new issus if you think it would be better.

Reply:
Thanks @dpordomingo! Will do so.